src/quo/components/drawers/bottom_actions/style.cljs

```diff
@@ -3,10 +3,9 @@
     [quo.foundations.colors :as colors]))
 
 (def buttons-container
-  {:flex-direction     :row
-   :justify-content    :space-around
-   :padding-vertical   12
-   :padding-horizontal 20})
```

Comment:
hmm will this not break other uses of this component? 🤔 The issue might be on the page itself actually. 
I would recommend using the regular button instead on that page instead as I think it's also fine to do. This bottom_actions is mostly for drawers as the ns suggests 👌

Reply:
done